diff --git a/node_modules/metro/src/blacklist.js b/node_modules/metro/src/blacklist.js
index 1a72099..bdeaf7d 100644
--- a/node_modules/metro/src/blacklist.js
+++ b/node_modules/metro/src/blacklist.js
@@ -12,15 +12,16 @@ var path = require('path');
 
 // Don't forget to everything listed here to `package.json`
 // modulePathIgnorePatterns.
-var sharedBlacklist = [
-/node_modules[/\\]react[/\\]dist[/\\].*/,
-
-/website\/node_modules\/.*/,
-
-/heapCapture\/bundle\.js/,
-
-/.*\/__tests__\/.*/];
 
+/**
+ * https://stackoverflow.com/a/58122821
+ */
+var sharedBlacklist = [
+  /node_modules[\/\\]react[\/\\]dist[\/\\].*/,
+  /website\/node_modules\/.*/,
+  /heapCapture\/bundle\.js/,
+  /.*\/__tests__\/.*/,
+];
 
 function escapeRegExp(pattern) {
   if (Object.prototype.toString.call(pattern) === '[object RegExp]') {
